.goreleaser.yml

```diff
@@ -34,6 +36,22 @@ archives:
         format: zip
     files: [include-only-the-binary*]
     wrap_in_directory: false
+brews:
+  -
+    name: ionosctl
+    ids:
+      - ionosctl
+    tap: 
+      owner: ionos-cloud
+      name: homebrew-ionosctl-tap
```

Comment:
@avirtopeanu-ionos I think `homebrew-ionos-cloud` is perfect, or `homebrew-ionos-tap` so that possibly more tools can be published under one umbrella.